src/amo/components/CollectionManager/index.js

```diff
@@ -55,6 +55,7 @@ type State = {|
   description?: string | null,
   name?: string | null,
   slug?: string | null,
+  userEnterSlug?: boolean | false,
```

Comment:
I would name this property: `customSlug`

Reply:
I really suck at naming :( 
will update :)